components/ui-shell/index.ts

```diff
@@ -0,0 +1,182 @@
+        },
+      ],
+    },
+    {
+      label: 'Documentation',
+      url: 'https://qiskit.org/documentation/',
+    },
+  ];
+
+  render() {
+    return html`
+      <bx-header aria-label="Qiskit" class="qiskit-header">
+        <qiskit-header-name href="https://qiskit.org/">
+          ${qiskitLogoIcon}
+        </qiskit-header-name>
+        <qiskit-header-nav menu-bar-label="Qikskit">
+          ${this._NAV_ITEMS.map((item) => {
+            if (item.children) {
+              return this._getHeaderMenu(item);
+            } else {
+              return this._getHeaderNavItem(item);
+            }
+          })}
+          ${this.variant === Variant.WITH_ACCOUNT
+            ? this._getAccountHeaderNavItem()
+            : null}
+        </qiskit-header-nav>
+        <qiskit-header-menu-button
+          button-label-active="Close menu"
+          button-label-inactive="Open menu"
+        >
+        </qiskit-header-menu-button>
+      </bx-header>
+
+      <bx-side-nav
+        aria-label="Main mobile navigation"
+        usage-mode="header-nav"
+        class="qiskit-side-nav"
+      >
+        <bx-side-nav-items>
+          ${this._NAV_ITEMS.map((item) => {
+            if (item?.children) {
+              return this._getSideNavMenu(item);
+            } else {
+              return this._getSideNavLink(item);
+            }
+          })}
+          ${this.variant === Variant.WITH_ACCOUNT
+            ? this._getAccountSideNavLink()
+            : null}
+        </bx-side-nav-items>
+      </bx-side-nav>
+    `;
+  }
+
+  private _getHeaderNavItem(item: NavItem) {
+    return html`<qiskit-header-nav-item href="${ifDefined(item?.url)}">
+      ${item?.label}
+    </qiskit-header-nav-item>`;
+  }
+
+  private _getHeaderMenu(menu: NavItem) {
+    return html`<qiskit-header-menu
+      menu-label="${menu?.label}"
+      trigger-content="${menu?.label}"
+    >
+      ${menu?.children?.map((item) => this._getHeaderMenuItem(item))}
+    </qiskit-header-menu>`;
+  }
+
+  private _getHeaderMenuItem(item: NavItem) {
+    return html`<qiskit-header-menu-item href="${ifDefined(item?.url)}">
+      ${item?.label}
+    </qiskit-header-menu-item>`;
+  }
+
+  private _getSideNavLink(item: NavItem) {
+    return html`<qiskit-side-nav-link href="${ifDefined(item?.url)}">
+        ${item?.label}
+      </qiskit-side-nav-link>
+      <bx-side-nav-divider></bx-side-nav-divider>`;
+  }
+
+  private _getSideNavMenu(menu: NavItem) {
+    return html`<qiskit-side-nav-menu title="${menu?.label}">
+        ${menu?.children?.map((item) => this._getSideNavMenuItem(item))}
+      </qiskit-side-nav-menu>
+      <bx-side-nav-divider></bx-side-nav-divider>`;
+  }
+
+  private _getSideNavMenuItem(item: NavItem) {
+    return html`<qiskit-side-nav-menu-item href="${ifDefined(item?.url)}">
+      ${item?.label}
+    </qiskit-side-nav-menu-item>`;
+  }
+
+  private _getAccountHeaderNavItem() {
+    return html`<qiskit-header-nav-item
```

Comment:
I think this could be updated to match what's currently in place, as well as what's in the [design file](https://www.figma.com/file/SOnOgCGQIWx4oAlSFhIsIX/1972-User-account?node-id=344%3A4530)

![Screen Shot 2022-09-02 at 6 20 45 AM](https://user-images.githubusercontent.com/6276074/188155460-7c90f900-f61d-43aa-b330-900d04288974.png)


I _do_ realize that what's currently implemented doesn't _exactly_ match what's designed... but I think as long as it has

- light background
- purple icon
- purple text

that should be acceptable IMO

Reply:
Resolved by https://github.com/Qiskit/web-components/pull/139/commits/23ff11e523adbbf9f92fd372561b8d12d67c091f

<img width="716" alt="Screen Shot 2022-09-02 at 9 15 49 AM" src="https://user-images.githubusercontent.com/8097306/188196331-5444f2c9-2313-4210-8695-0cf825573bd9.png">